src/pages/SocialGraph.vue

```diff
@@ -0,0 +1,133 @@
+        </tr>
+      </table>
+      <td>{{ c.totalValue }} LIKE</td>
+    </tr>
+  </table>
+  <p v-else>No response</p>
+</template>
+
+<script>
+import axios from 'axios';
+import {
+  INDEXER, IGNORE_LIST,
+} from '../config';
+import { getClass, getMetadata } from '../utils/proxy';
+
+export default {
+  name: 'NftSocialGraph',
+  data() {
+    return {
+      type: 'collector',
+      account: 'like13f4glvg80zvfrrs7utft5p68pct4mcq7t5atf6',
+      responseType: '',
+      response: [],
+      ignoreList: IGNORE_LIST,
+    }
+  },
+  methods: {
+    async load() {
+      if (this.type === 'collector') {
+        const res = await axios.get(`${INDEXER}/likechain/likenft/v1/collector`, {
+          params: {
+            creator: this.account,
+            reverse: true,
+          },
+        });
+        this.response = res.data.collectors;
+        this.responseType = 'Collectors';
+        this.response = await this.aggregate(this.response);
+      }
+      if (this.type === 'creator') {
+        const res = await axios.get(`${INDEXER}/likechain/likenft/v1/creator`, {
+          params: {
+            collector: this.account,
+            reverse: true,
+          },
+        });
+        this.response = res.data.creators;
+        this.responseType = 'Creators';
+        this.response = await this.aggregate(this.response);
+      }
+    },
+    async aggregate(accounts) {
+      const promises = [];
+      const newAccounts = [];
+      accounts.filter(a => !this.ignoreList.includes(a.account)).forEach((a) => {
+        const account = {
+          account: a.account,
+          collections: [],
+          count: a.count,
+          totalValue: 0,
+        };
+        a.collections.forEach(
+          ({ iscn_id_prefix: iscnIdPrefix, class_id: classId, count }) => {
+            let collection = {
+              iscnIdPrefix,
+              classId,
+              count,
+            }
+            promises.push(
+              getClass(classId)
+              .then((res) => {
+                const {
+                  lastSoldPrice: price,
+                } = res.data;
+                collection.price = price;
+                collection.totalValue = count * price;
+                account.totalValue += count * price;
+                return getMetadata(classId)
+              })
+              .then((res) => {
+                collection = {
+                  ...collection,
+                  ...res.data,
+                }
+              })
+              .catch((err) => {
+                console.error(err.message, iscnIdPrefix, classId);
+                collection.price = 0;
+                collection.totalValue = 0;
+              })
+              .finally(() => {
+                account.collections.push(collection)
+              })
+            );
+          },
+        );
+        newAccounts.push(account);
+      });
+      await Promise.all(promises);
```

Comment:
I think this part would be much cleaner if we are able to use store `elegantly` , but it's optionally and depends if you have interested, if not we can keep it this way.